### PR TITLE
Add korean translation

### PR DESCRIPTION
**Pull request by eunsucking**
Add Korean translation.
Review please @marocchino @yous @MJayJang 😵

### CODE REVIEW

File: exercises/cat_annotation/problem.ko.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# 고양이 사진과 다각형
+
+새로운 기능을 추가해 볼까요? 이미지 위에 다각형 영역을 그려 봅시다.
+
+이 기능은 어떤 물체에 표시할 때나, 지도 위에 일정 지역을 나타낼 때 유용합니다.
+
+`index.html` 파일의 `<body>` 부분에 다음 코드 한 줄을 넣어 보세요. 이곳은 사진이 위치할 빈 `div` 요소입니다. **조심하세요.** 이 코드는 `<script>` 태그보다 **이전에** 위치해야 합니다. 왜냐하면 `div` 요소가 생성된 뒤에 `<script>`내의 코드가 실행되기 위해서입니다.
```
Comment (yous) [inline exercises/cat_annotation/problem.ko.md]: How about '태그 \*\*이전에\*\*'?

File: exercises/cat_annotation/problem.ko.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+```
+<div id='visualization'></div>
+```
+
+다음은 `index.js`를 수정해 볼까요? 맨 처음 줄에 고양이 사진을 require를 이용해서 변수에 저장해 봅시다.
```
Comment (yous) [inline exercises/cat_annotation/problem.ko.md]: How about 'require를 이용해서 고양이 사진을'?

File: exercises/cat_picture/problem.ko.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+이제 새로운 기능들을 추가할 작업대가 만들어졌습니다. 그럼 첫 번째 기능을 추가해볼까요? 고양이 사진을 그려봅시다.
+
+기능을 추가하기 위해 아예 처음부터 만드는 방법도 있습니다만, 여타 다른 프로그래밍 작업처럼, 누군가가 우리가 필요로 하는 기능을 이미 만든 경우가 많습니다. Node.js는 개발자가 자신이 작성한 프로그램을 패키지화해서 재사용을 쉽게 하도록 해줍니다. 또한 `npm`이라는 프로그램을 제공하여 그러한 프로그램(Node는 이걸 "modules"라고 부릅니다)들을 만들고, 내려받고, 설치하게 해줍니다. R을 사용해보신 적이 있다면 `npm`은 `install.packages()`와 비슷하다고 생각하시면 됩니다.
+
+자 이제 본론으로 돌아와서 아름다운 고양이 사진을 앱에 추가해봅시다. 그러려면 `cat-picture`라는 모듈을 npmjs.org로부터 설치할 필요가 있습니다. 이 모듈은 웹 페이지에 고양이 사진을 추가하는 단순한 프로그램입니다.
+
+`cat-picture` 모듈을 설치하기 위해 `npm`이라는 명령어를 사용할것입니다. 이 작업을 하기 전에, 지금 만들고 있는 앱은 무엇을 하는 프로그램인지에 대해 Node.js에 알려줘야 합니다. 가장 좋은 방법은 실제로 _당신이 직접_ 모듈을 작성하는 것입니다.
+
+모듈을 작성하려면 `package.json`이라는 파일을 만들면 됩니다. 이 파일에 우리가 만든 모듈에 대해서, 그리고 우리의 모듈이 의존하고 있는 모듈들의 리스트를 작성해야 합니다. 고맙게도 `npm`은 이 작업을 우리 대신에 해줍니다!
```
Comment (yous) [inline exercises/cat_picture/problem.ko.md]: 우리 대신에 → 우리 대신

File: exercises/hello_world/problem.ko.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
+# 헬로 월드
+
+간단한 electron 앱을 만들어 봅시다.
+
+먼저, 새로운 폴더를 만들고 그 폴더에서 앞으로의 작업을 진행해 주세요.
+
+텍스트 에디터(TextEdit, Notepad 등등)를 사용해서 방금 만든 폴더에 `index.html`을 만들고, 다음 내용을 작성하세요.
```
Comment (yous) [inline exercises/hello_world/problem.ko.md]: 등등 → 등

It's just a preference, though.

File: i18n/ko.json [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+      "success": "현재 앱 폴더와 폴더 내의 파일들이 정상적입니다.",
+      "fail": "현재 앱 폴더는 뭔가가 부족합니다.",
+      "cant_read": "파일에 접근할 수 없습니다.",
+      "invalid_html": "html 파일에 문제가 있습니다.",
+      "missing_script_tag": "script 태그를 찾을 수 없습니다.",
+      "missing_body_tag": "body 태그를 찾을 수 없습니다.",
+      "script_not_in_body": "script 태그가 body 태그 내에 존재하지 않습니다."
+    },
+    "hello_world": {
+      "success": "앱이 정상입니다.\n\n`electron app.js`을 실행하면 Electron 세계로부터의 환영 메시지가 출력될 것입니다.",
```
Comment (yous) [inline i18n/ko.json]: \`electron app.js\`를

**Review by yous — COMMENTED** [review]
Thanks for your effort!

File: exercises/cat_annotation/problem.ko.md (hunk 1 above)
Comment (marocchino) [reply exercises/cat_annotation/problem.ko.md]: avoid use kanji
이전에 -> 앞에

Comment (marocchino) [reply exercises/cat_annotation/problem.ko.md]: how about this?
```diff
-왜냐하면 ~~ 실행되기 위해서입니다.
+그래야 ~~ 실행됩니다.
```

File: exercises/cat_annotation/problem.ko.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+
+```
+var picture = require('cat-picture')
+```
+
+실제 필요한 것은 고양이 사진 데이터이므로, 변수에 저장한 뒤에 페이지에 표시된 사진을 지워주세요.
```
Comment (marocchino) [inline exercises/cat_annotation/problem.ko.md]: > We actually just want the source data for the picture, so get it, then remove the image from the page.

```diff
-실제 필요한 것은 고양이 사진 데이터이므로, 변수에 저장한 뒤에 페이지에 표시된 사진을 지워주세요.
+사실 필요한 것은 고양이 사진의 소스 데이터뿐이니, 변수에 저장한 뒤에 사진을 페이지에서 지웁시다.
```

File: exercises/cat_annotation/problem.ko.md [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+
+```
+var src = picture.src
+picture.remove()
+```
+
+이제 다각형을 그려주는 모듈을 설치해 봅시다. `npm install lightning-image-poly --save`를 입력해 주세요.
+
+`package.json`에 새로운 줄이 추가된 것을 확인해 보세요.
```
Comment (marocchino) [inline exercises/cat_annotation/problem.ko.md]: > Double check that your package.json has a new entry!

```diff
- `package.json`에 새로운 줄이 추가된 것을 확인해 보세요.
+ `package.json`에 새로운 항목이 추가되었는지 다시 한번 확인하세요.
```

File: exercises/cat_annotation/problem.ko.md [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+
+다음 코드를 추가해서 설치한 모듈을 사용해봅시다.
+
+```
+var image = require('lightning-image-poly')
+```
+
+이제 이 모듈로 이미지를 저장한 변수를 사용해서, 이미지를 그려봅시다.
```
Comment (marocchino) [inline exercises/cat_annotation/problem.ko.md]: Then render the image data we stored above by creating a new image visualization.

```diff
- 이제 이 모듈로 이미지를 저장한 변수를 사용해서, 이미지를 그려봅시다.
+ 위에서 변수에 저장한 이미지 데이터를 사용해,  visualization `image`를 새로 만들어 이미지를 그려봅시다.
```

File: exercises/hello_world/problem.ko.md (hunk 4 above)
Comment (eunsucking) [reply exercises/hello_world/problem.ko.md]: It looks better writing '등' once. 👍

**Review by yous — APPROVED** [review]

**Comment by eunsucking** [comment]
Thanks for proofreading! @marocchino @yous 😍

**Review by marocchino — APPROVED** [review]

**Comment by MJayJang** [comment]
👍👍👍

**Comment by zeke** [comment]
Thanks everyone! Is this ready to go?

**Comment by eunsucking** [comment]
We're done! Merge please. :grinning:

**Pull request merged**